assets/js/src/common/button/button.tsx

```diff
@@ -12,6 +12,7 @@ type Props = {
   iconEnd?: JSX.Element,
   onClick?: () => void,
   href?: string,
+  type?: string,
```

Comment:
This should be `'a' | 'button'` I guess. Also, a default should be in the component header below (but all defaults seem to be missing, actually).

Reply:
Hmm, looking further, it seems to be unused...

Reply:
Good catch, I automatically assumed (knowing how many components work), that all HTML attributes are automatically used as part of `...attributes` prop - but that's not used in `<Button>` component. Added. 

And this `type` is used for [HTML button type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type), not to differentiate what element to use - added possible values, so it's clear.